src/hooks/useCatMessages.tsx

```diff
@@ -144,38 +198,64 @@ export default function useCatMessages(propertyFeatures: PropertyFeatures) {
           id: "visualize",
           text: "Lets visualize how your life could look like here....",
           catImageNumber: 1,
-          continueButtonText: "Start Simulation",
-          continueButtonVariant: "outline",
+          options: [
+            {
+              text: "Start simulation",
+              action: async () => {
+                if (ageRange && gender && stageOfLife) {
+                  const res = await getLifeSimulation(propertyId, ageRange, gender, stageOfLife);
+                  setTourIteinerary(res);
```

Comment:
getting cors here in the preview here

Reply:
solved for the preivew by overwriting the base path URL of API to be this specific preview url